app/services/prison_visits/api.rb

```diff
@@ -59,15 +59,19 @@ def get_slots(
       prison_id:, prisoner_number:, prisoner_dob:, use_nomis_slots: false
     )
       response = @client.get(
-        '/available_slots',
+        '/slots',
         params: {
-          prison_id: prison_id,
-          prisoner_number: prisoner_number,
-          prisoner_dob: prisoner_dob,
-          use_nomis_slots: use_nomis_slots
+          prison_id: prison_id, prisoner_number: prisoner_number,
+          prisoner_dob: prisoner_dob, use_nomis_slots: use_nomis_slots,
+          start_date: Time.zone.today, end_date: 28.days.from_now.to_date
```

Comment:
Do we need to pass the date range here? 

Most prisons are 28 days, but some have shorter booking windows, however I don't see a use case for the client to request different date ranges. I think it would be best to take out the date range params and treat this as an implementation detail on the server.